sale_quotation_template_product_multi_add/tests/test_sale.py

```diff
@@ -0,0 +1,33 @@
+import odoo.tests.common as common
+from odoo.tests import tagged
+
+
+@tagged("post_install")
```

Comment:
Why do you need this ?

Reply:
I thought it's required to pass tests. Removed it.

Reply:
If you saw something about that, that's only when using Odoo's AccountInvoiceCommon test class.